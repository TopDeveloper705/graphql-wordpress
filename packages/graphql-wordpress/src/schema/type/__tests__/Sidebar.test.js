import SidebarType from 'type/Sidebar';

// eslint-disable-next-line no-underscore-dangle
const fields = SidebarType._typeConfig.fields;

describe('Test Sidebar type', () => {
  test('Test name', () => {
    expect(SidebarType.name).toMatchSnapshot();
  });

  test('Test fields', () => {
    expect(Object.keys(SidebarType.getFields())).toMatchSnapshot();
  });

  test('Test resolve name', () => {
    const classname = fields.classname.resolve({ class: 'cool-widget' });
    expect(classname).toMatchSnapshot();
  });
});
