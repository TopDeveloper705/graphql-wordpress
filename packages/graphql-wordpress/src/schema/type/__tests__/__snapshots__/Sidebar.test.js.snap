// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Sidebar type Test fields 1`] = `
Array [
  "id",
  "name",
  "description",
  "classname",
  "beforeWidget",
  "afterWidget",
  "beforeTitle",
  "afterTitle",
  "widgets",
]
`;

exports[`Test Sidebar type Test name 1`] = `"Sidebar"`;

exports[`Test Sidebar type Test resolve name 1`] = `"cool-widget"`;
